### PR TITLE
add `uuid-utils`

### PR DESCRIPTION
**Pull request by moritzwilksch**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/uuid-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/uuid_utils-{{ version }}.tar.gz
+  sha256: 7e828735e8b84186e37ba0942f8873c29fb3d175eaeb5bb0b5ce7e10464e7858
+
+build:
+  noarch: python
```
Comment (xhochy) [inline recipes/uuid-utils/meta.yaml:13]:
```suggestion
```

File: recipes/uuid-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
```
Comment (xhochy) [inline recipes/uuid-utils/meta.yaml:17]:
```suggestion
requirements:
  build:
    - {{ compiler('rust') }}
```

**Review by xhochy — CHANGES_REQUESTED** [review]
If this is rust, it should be a compiled package

**Comment by moritzwilksch** [comment]
Thanks, I still need to add the `build.sh` and `build.bat` then and collect the licenses of the rust dependencies using `cargo-bundle-licenses` right?

File: recipes/uuid-utils/bld.bat [hunk 3]
```diff
@@ -0,0 +1,19 @@
+REM Create temp folder
+mkdir tmpbuild_%PY_VER%
+set TEMP=%CD%\tmpbuild_%PY_VER%
+REM Bundle all downstream library licenses
+cargo-bundle-licenses ^
+    --format yaml ^
+    --output %SRC_DIR%\THIRDPARTY.yml ^
+    || goto :error
+REM Use PEP517 to install the package
+set RUSTFLAGS="--cfg uuid_unstable"
```
Comment (xhochy) [inline recipes/uuid-utils/bld.bat:10]: Quoting is different on Windows
```suggestion
set "RUSTFLAGS=--cfg uuid_unstable"
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by moritzwilksch** [comment]
@conda-forge/help-python, ready for review!

File: recipes/uuid-utils/build.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+# Bundle all library licenses
+cargo-bundle-licenses \
+  --format yaml \
+  --output ${SRC_DIR}/THIRDPARTY.yml
+
+# Apply PEP517 to install the package
+
+export RUSTFLAGS="--cfg uuid_unstable"
+maturin build --release -i $PYTHON
+ls -la
```
Comment (xhochy) [inline recipes/uuid-utils/build.sh:14]:
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]
Please remove debug code.

**Review by xhochy — APPROVED** [review]

**Pull request merged**